xbdd-web/src/containers/ScenarioListViewContainer.jsx

```diff
@@ -0,0 +1,58 @@
+import React, { Component } from 'react';
```

Comment:
Probably remove this file as well as its references in the app :) as it will be pushed as part of another PR